README.md

```diff
@@ -84,7 +95,11 @@ Define a global variable instructor and set it equal to your Squad Instructor's
 
 Your Answer:
 ```js
-// write code here
+var instructor = "Becky";
```

Comment:
Close! What I was getting at with this particular question is the following, essentially writing two different variables, one global and one local:
```
// global variable
instructor = "Becky";
// local variable
var instructor = "Becky";
```